docs/model/simulation_outputs.rst

```diff
@@ -79,41 +81,45 @@ The capital recovery factor (CRF) is used to calculate the present value of the
 
 .. include:: outputs/levelized_costs_of_electricity_equivalent.inc
 
-As a sector-coupled system connects energy vectors, not the costs associated to each individual energy carrier but the overall energy costs should be minimized. Therefore, we propose a new KPI: The levelized costs of energy (LCOEnergy) aggregates the costs for energy supply and distributes them over the total energy demand supplied, which is calculated by weighting the energy carriers by their energy content. To determine the weighting factors of the different energy carriers, we reference the method of gasoline gallon equivalent (GGE) [12], which enables the comparison of alternative fuels. Instead of comparing the energy carriers of an MES to gasoline, we rebase the factors introduced in [12] onto the energy carrier electricity, thus proposing a unit Electricity Equivalent (ElEq). The necessary weights are summarized in Table 1. With this, we propose to calculate LCOEnergy based on the annual energy demand and the systems annuity, calculated with the CRF, as follows:
+A sector-coupled system connects energy vectors, not the costs associated to each individual energy carrier. However, the overall energy costs should be minimized. Therefore, we define a new KPI: The levelized costs of energy (LCOEnergy), which aggregates the costs for energy supply and distributes them over the total energy demand supplied. The latter is calculated with weighting factors for each energy carriers. To determine the weighting factors of the different energy carriers, we reference the method of gasoline gallon equivalent (GGE) `[12] <https://epact.energy.gov/fuel-conversion-factors>`_, which enables the comparison of alternative fuels. Instead of comparing the energy carriers of an MES to gasoline, we rebase the factors introduced in `[12] <https://epact.energy.gov/fuel-conversion-factors>`_ onto the energy carrier electricity, thus proposing a unit Electricity Equivalent (ElEq). The necessary weights are summarized in Table 1. With this, we propose to calculate LCOEnergy based on the annual energy demand and the systems annuity, calculated with the CRF, as follows:
```

Comment:
```suggestion
A multi-vector energy system connects energy vectors into a joined, energy system. The optimization objective of the MVS then is to minimize the overall energy costs, without distinguising the different sectors. This sector-coupled energy system is then designed to have an optimial, joined operation. With other systems, the costs associated to each individual energy vector would be used to calculate the levelized costs of energy (LCOEnergy). With the multi-vector system, this could lead to distorted costs - for example if there is a lot of PV (electricity sector), which in the end is only supplying an electrolyzer (H2 sector). The LCOE of electricity would thus turn out to be very high, which could be considered unfair as the electricity from PV is solely used to provide the H2 demand. 
Therefore, we define the :ref:`attributed costs of each energy vector <attributed_costs>`, to determine how much of the overall system costs should be attributed to one sector, depending on the energy demand it has compared to the other sectors. To be able to compare the demands of different energy carriers, :ref:`energy carrier weighting <energy_carrier_weighting>` is applied. 
We then define the the levelized costs of energy (LCOEnergy), based on the annuity of the attributed costs and the demand of one energy sector.  
```
How about this? The first part about the attributed costs could also be moved further up. The details about energy carrier weighting are already explained here: https://multi-vector-simulator.readthedocs.io/en/latest/model/assumptions.html#weighting-of-energy-carriers